src/BenchmarksClient/Workers/WorkerFactory.cs

```diff
@@ -15,10 +15,10 @@ static public class WorkerFactory
         static WorkerFactory()
         {
             // Wrk
-            Workers[Worker.Wrk] = clientJob => new WrkWorker(clientJob);
+            Workers[Worker.Wrk] = clientJob => new WrkWorker();
```

Comment:
I assume we can just remove the `clientJob` argument and pass it in `Start` and `Stop`

Reply:
Yeah, we don't need to pass the clientJob into the worker constructors anymore